### PR TITLE
[BUU] Enqueue actions to perform at end of reflex

### PR DESCRIPTION
**Pull request by dacook**
#### What? Why?
I'm trying to better understand StimulusReflex and doing sort of a health check.

I don't think this will fix https://github.com/openfoodfoundation/openfoodnetwork/issues/12020, but it means sticking a bit closer to convention, and I think things will run slightly smoother.

#### What should we test?
Just a quick run through of the v3 Bulk Edit Products screen, to test the responsiveness:

- Using filters 
- Using different page sizes
- Updating a product

Previously, the success message would sometimes appear before the loading spinner had finished. Now I think it should all happen at the same time.

#### Release notes

Changelog Category (reviewers may add a label for the release notes):

- [ ] User facing changes
- [ ] API changes (V0, V1, DFC or Webhook)
- [x] Technical changes only
- [ ] Feature toggled

The title of the pull request will be included in the release notes.

### CODE REVIEW

File: app/reflexes/products_reflex.rb [hunk 1]
```diff
@@ -136,9 +136,8 @@ def render_products_form_with_flash
     )
     morph :nothing
 
-    # dunno why this doesn't work.
-    # morph "#products-form", render(partial: "admin/products_v3/table",
-    #                locals: { products: products })
+    # dunno why this doesn't work. The HTML stops after the first `<col>` element, wtf?!
+    # morph "#products-form", render(partial: "admin/products_v3/table", locals:)
```
Comment (mkllnk) [inline app/reflexes/products_reflex.rb:140]: Do you know if this question is about morph vs replace or is it about cable_ready vs stimulus_reflex?

**Review by mkllnk — APPROVED** [review]

File: app/reflexes/products_reflex.rb (hunk 1 above)
Comment (dacook) [reply app/reflexes/products_reflex.rb:140]: I don't know. When I wrote the comment I assume `morph` and `replace` should do the same thing, but I realise now those words mean different things. Time to read more of the manual...

**Review by rioug — APPROVED** [review]
Nice find on the order of operations !

**Comment by RachL** [comment]
@dacook LGTM but there are conflicts that must be solved before merging. Moving back to indev.

**Comment by dacook** [comment]
Conflicts merged, and there are changes to test, so merging!

**Pull request merged**